packages/venia-concept/package.json

```diff
@@ -26,7 +26,8 @@
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook:build": "build-storybook -c .storybook -o storybook-dist",
     "validate-queries": "node ./validate-queries.js",
-    "watch": "webpack-dev-server --progress --color --env.mode development"
+    "watch": "webpack-dev-server --progress --color --env.mode development",
+    "watch:docker": "webpack-dev-server --progress --color --env.mode development --host pwa-docker.localhost"
```

Comment:
I don't think a docker specific script is necessary, just pass runtime arguments

`yarn watch -- --host pwa-docker.localhost`

This might allow you to pull this value from `.env.docker` so devs don't have to manually make sure this value is configured correctly in all places.

Reply:
this was one thing I really wanted to pull in from env vars so that all the configuration is only in the `.env.docker` file. I will look into this and hopefully get it to pull in 🙏

Reply:
this is fixed now!